src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -900,4 +900,16 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_Packages" xml:space="preserve">
     <value>Packages</value>
   </data>
+  <data name="Status_Canceled" xml:space="preserve">
+    <value>Package results loading cancelled</value>
+  </data>
+  <data name="Status_ErrorOccurred" xml:space="preserve">
+    <value>Error occurred when loading package results</value>
```

Comment:
Is "package results" a good way to describe this to the user? "An error occurred while loading packages" is my first thought.

Reply:
Applied your suggestion